__tests__/Config.test.js

```diff
@@ -92,12 +92,12 @@ describe('Config', () => {
       expect(Config.sinopiaUrl).toEqual('https://sinopia.foo')
     })
 
-    it('sinopia server url overrides static value', () => {
+    it('USE_FIXTURES overrides static value', () => {
```

Comment:
I'm not sure I get this change here. This is about the server URL, not the spoofs/fixtures, right?

Reply:
oops.  fixing.